src/daemon/daemon.cpp

```diff
@@ -1949,69 +1890,69 @@ void mp::Daemon::create_vm(const CreateRequest* request, grpc::ServerWriter<Crea
             delete prepare_future_watcher;
         });
 
-    prepare_future_watcher->setFuture(QtConcurrent::run([this, server, request, name,
```

Comment:
It looks to me that clang-format did it's thing here until line 2014/1955, right?  I think it did which is unfortunate because it really pollutes this PR :frowning_face:

Reply:
Yep. I don't think it is worth fighting it, but could you sanity check and run your clang-format against it? Just in case (I devel on bionic)

Reply:
I can't seem to get anything different via `clang-format`, so I'll just resolve this.